snooty.toml

```diff
@@ -23,4 +23,5 @@ api = "https://mongodb.github.io/mongo-java-driver/{+version+}/apidocs"
 rs-docs = "https://www.reactive-streams.org/reactive-streams-1.0.4-javadoc/org/reactivestreams"
 driver-source-gh = "https://github.com/mongodb/mongo-java-driver"
 string-data-type = "``String``"
-bool = "``boolean``"
\ No newline at end of file
+bool = "``boolean``"
+pr = "Project Reactor"
```

Comment:
S: since you made this constant, can you replace other instances of it that are in the docs?

Reply:
Rea and I discussed this over slack; added this to my clean up ticket